pages_builder/pages/big-stat.yml

```diff
@@ -0,0 +1,9 @@
+pageTitle: Big statistic
```

Comment:
Similar to the comment below, could you call this `statistics.yml`. Or change the others to `big-stat.yml`. Just so they're consistent.